src/index.html

```diff
@@ -1,14 +1,54 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="header__a">
+        <img src="/src/images/logo.png" alt="logo_moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones">Smartphones</a>
```

Comment:
Need to add classes for the tag a
```suggestion
            <a class="nav__link" href="#smartphones">Smartphones</a>
```